.travis.yml

```diff
@@ -136,8 +136,8 @@ after_success:
         echo "Finished mvn clean deploy for $TRAVIS_BRANCH";
         pushd .;
         cd modules/openapi-generator-gradle-plugin;
-        ./gradlew -Psigning.keyId="$SIGNING_KEY" -Psigning.password="$SIGNING_PASSPHRASE" -Psigning.secretKeyRingFile="${TRAVIS_BUILD_DIR}/sec.gpg" -PossrhUsername="${SONATYPE_USERNAME}" -PossrhPassword="${SONATYPE_PASSWORD}" uploadArchives --no-daemon;
-        echo "Finished ./gradlew uploadArchives";
+        ./gradlew -Psigning.keyId="$SIGNING_KEY" -Psigning.password="$SIGNING_PASSPHRASE" -Psigning.secretKeyRingFile="${TRAVIS_BUILD_DIR}/sec.gpg" -PossrhUsername="${SONATYPE_USERNAME}" -PossrhPassword="${SONATYPE_PASSWORD}" publishPluginMavenPublicationToNexusRepository --no-daemon;
+        echo "Finished ./gradlew publishPluginMavenPublicationToNexusRepository";
```

Comment:
We need to see how this goes, but I guess that calling the task `closeAndReleaseRepository` would also be necessary. (the maven nexus plugin is doing something like this)

Reply:
This is actually the SNAPSHOT path (master), but your comment made me realize that I had completely missed updating the `uploadArchives` in the following conditional block which is used for releases. I believe the releases branch is the only one that requires closing.